assets/quick-add-bulk.js

```diff
@@ -24,7 +24,10 @@ if (!customElements.get('quick-add-bulk')) {
 
       connectedCallback() {
         this.cartUpdateUnsubscriber = subscribe(PUB_SUB_EVENTS.cartUpdate, (event) => {
-          if (event.source === 'quick-add') {
+          if (
+            event.source === 'quick-add' ||
+            (event.cartData.items && !event.cartData.items.some((item) => item.id === this.dataset.index))
+          ) {
             return;
           }
           // If its another section that made the update
```

Comment:
Good catch!

Reply:
I still see those errors ([recording](https://screenshot.click/28-35-n1gv6-pbbld.mp4)) seems to be due to the fact that on: 
```html
<div class="collection{% if section.settings.filter_type != 'vertical' %} page-width{% endif %}">
```
we don't have a `data-id="{{ section.id }}"`

But that wasn't changed on this PR originally 🤷 and doesn't seem to prevent adding to cart

Reply:
That is because `data.id` lives in the `ul` element inside of it 🤔 I will create a follow up

Although I cant seem to reproduce it - https://screenshot.click/28-30-m25f3-8he2d.mp4